server/controllers/ActivityType.controller.js

```diff
@@ -16,46 +17,63 @@ exports.getAllActivityTypes = (req, res) => {
 	});
 }
 
-exports.createNewActivityType = (req, res) => {
-	const activityTypeName = req.body.activityTypeName;
-	const newActivityType = new ActivityType({ name: activityTypeName });
-	newActivityType.save(
-		(err) => {
-				if (err) {
-					return res.status(400).send({message: `Failed to save new activity type. ${err}`});
+exports.create = (req, res) => {
+	if (JWTVerifier.verifyStaffToken(req.body.token, process.env.STAFF_SECRET_KEY)) {
```

Comment:
This function can be called as middleware where the route is defined. More about middleware functions: https://reflectoring.io/express-middleware/ 
icard example: https://github.com/UofA-Blueprint/ICard/blob/develop/backend/src/routes/Students.js#L63 (notice how verifyApiKey is called before the actual getStudent controller is called in the route definition).

Reply:
If you decide to make this change, please also change all other endpoints accordingly.

Reply:
Done